include/dungeon_entity.h

```diff
@@ -215,7 +215,8 @@ struct DungeonEntityData
     /* 0x16C */ struct Position targetPosition;
     /* 0x170 */ struct Position posPixel;
     u32 unk174;
-    u8 fill178[0x184 - 0x178];
+    u8 fill178[0x17A - 0x178];
+    u16 mimicMoveIDs[5]; // All moves that Mimic has copied (not sure on size...)
```

Comment:
There are only 4 move slots, so the size is likely 4 (`MAX_MON_MOVES`), leaving an unknown after this array.

Reply:
Testing locally, it aligns/matches with this and no unknown so will probably run with this
```suggestion
    u16 mimicMoveIDs[MAX_MON_MOVES]; // All moves that Mimic has copied (not sure on size...)
```

Reply:
Oh nice, that's probably correct then.